cypress/integration/focus-trap-demo.spec.js

```diff
@@ -19,7 +19,7 @@ describe('<FocusTrap> component', () => {
 
       // activate trap
       cy.get('@testRoot')
-        .findByRole('button', { name: 'activate trap' })
+        .findByRole('button', { name: "activate trap for 'defaults' demo" })
```

Comment:
Is there a way to give Cypress a regex here like `^activate trap` so that we could keep the test code more simple and consistent from test to another, while still being able to keep different labels on buttons in the HTML? I feel like this would keep test code simpler to copy/paste and then tweak, like you can do today provided you follow pre-established patterns.

Reply:
That's a great suggestion. I've updated this to `.findByRole('button', { name: /^activate trap/ })` and it works beautifully.